src/components/tabs/tab-group.spec.ts

```diff
@@ -47,44 +49,72 @@ describe('MdTabGroup', () => {
       checkSelectedIndex(2);
     });
 
-    it('should cycle through tab focus with focusNextTab/focusPreviousTab functions', () => {
+    it('should cycle through tab focus with focusNextTab/focusPreviousTab functions',
```

Comment:
Looks like nothing is currently testing that the event object has the correct data.